rlp_txn_v2/generalities/transaction_decoding.tex

```diff
@@ -0,0 +1,35 @@
+We define the following shorthands
+\[
+	\left\{ \begin{array}{lcl}
+		\locTransactionTypeSum _{i} & \define &
+		\left[ \begin{array}{cl}
+			+ & \typeZeroTx _{i} \\
+			+ & \typeOneTx  _{i} \\
+			+ & \typeTwoTx  _{i} \\
+		\end{array} \right]
+		\vspace{2mm} \\
+		\locWeightedTransactionTypeSum _{i} & \define &
+		\left[ \begin{array}{clcl}
+			+ & 0 & \!\!\!\cdot\!\!\! & \typeZeroTx _{i} \\
+			+ & 1 & \!\!\!\cdot\!\!\! & \typeOneTx  _{i} \\
+			+ & 2 & \!\!\!\cdot\!\!\! & \typeTwoTx  _{i} \\
+		\end{array} \right]
+	\end{array} \right.
+\]
+We impose:
+\begin{enumerate}
+	\item
+		\typeZeroTx{}, \typeOneTx{} and \typeTwoTx{} are \textbf{binary transaction constant} \quad (\sanityCheck)
+	\item
+		we impose that $\locTransactionTypeSum _{i} = \locPhaseFlagSum _{i}$
```

Comment:
AFAICT nothing preveted us until now to have

```rust
TX_TYPE_0 = 0
TX_TYPE_1 = 0
TX_TYPE_2 = 0
TX_TYPE_3 = 0 // later
TX_TYPE_4 = 0 // later
```
if `TX_TYPE = 0` (i.e. legacy transaction.)